src/modules/songs/components/SongNotes/SongNotes.tsx

```
@@ -0,0 +1,167 @@
+        <button
+          className={cn("hover:cursor-pointer", {
+            "p-[9px]": !isEditingNotes,
+          })}
+          onClick={() => setIsEditingNotes(true)}
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              e.preventDefault();
+              setIsEditingNotes(true);
+            }
+          }}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)